docker/yum-autodetect/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM centos:latest
+
+WORKDIR /ahab-docker
+
+RUN yum -y install epel-release which python3-pip
+
+COPY ahab .
+
+# Spit out these just for easier debugging
+RUN yum list installed
+
+RUN yum list installed | ./ahab chase
```

Comment:
Trivial: newline

Reply:
so to be clear, you would like to see a newline after the RUN command with ahab in it or you would like to see it removed between the RUN statements??